test/lib/cql_test_env.cc

```diff
@@ -404,6 +405,10 @@ class single_node_cql_env : public cql_test_env {
         return _task_manager;
     }
 
+    virtual sharded<locator::shared_token_metadata>& get_shared_token_metadata() override {
```

Comment:
I think this is needed only for perf_simple_query, why even audit code is there? seems like we don't need it

Reply:
maybe __builtin_unreachable() then if it's not used?